net/ddns-go/file/ddns-go.init

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh /etc/rc.common
+#
+# Copyright (C) 2021-2022  sirpdboy  <herboy2008@gmail.com> https://github.com/sirpdboy/luci-app-ddns-go 
+#
+# This file is part of ddns-go .
+# 
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+
+START=99
+USE_PROCD=1
+
+PROG=/usr/bin/ddns-go
+CONFDIR=/etc/ddns-go
+CONF=/etc/ddns-go/ddns-go-config.yaml
+
+get_config() {
+        config_get_bool enabled $1 enabled 1
+	config_get_bool logger $1 logger 1
+	config_get port $1 port 9876
+}
+ddns-go_prepare() {
+	pgrep -f /usr/bin/ddns-go | xargs kill -9 >/dev/null 2>&1 
+	logger -t ddns-go -p warn "ddns-go is stop."
+}
+
+init_yaml(){
+	[ -d $CONFDIR ] || mkdir -p $CONFDIR 2>/dev/null
+	cat /usr/share/ddns-go/ddns-go-default.yaml  > $CONF
+}
+
+stop_service() {
+	ddns-go_prepare
```

Comment:
If you are using procd to manage the service, why do you need to manually kill it?